src/components/utility/Button/Button.module.css

```diff
@@ -1,70 +1,22 @@
-:root {
-  --primary-white-color: #fff;
-  --primary-hover-color: #002529;
-  --primary-disabled-color: #666666;
-  --primary-disabled-border-color: #cccccc;
-  --secondary-hover-color: #79c8c7;
-  --secondary-disabled-color: #374041;
-  --tertiary-color: #012f34;
-  --tertiary-disabled-color: #666666;
-  --tertiary-disabled-border-color: #a0a0a0;
-}
 .button {
   padding: 10px 20px;
-  border: none;
+  border: 2px solid var(--button-border-color);
   border-radius: 24px;
   cursor: pointer;
-  font-size: 18px;
-  line-height: 28px;
-  font-style: normal;
-  transition: background-color 0.3s, color 0.3s;
-}
-
-/* Primary Button */
-.buttonPrimary {
-  background-color: transparent;
-  color: var(--primary-white-color);
-  border: 2px solid var(--primary-white-color);
-}
-
-.buttonPrimaryHover:hover {
-  background-color: var(--primary-white-color);
-  color: var(--primary-hover-color);
-}
-
-.buttonPrimaryDisabled:disabled {
+  font-size: 1rem;
+  transition: background-color 0.3s, color 0.3s, border-color 0.3s;
   background-color: transparent;
-  color: var(--primary-disabled-color);
-  border: 2px solid var(--primary-disabled-border-color);
-}
-
-/* Secondary Button */
-.buttonSecondary {
-  background-color: #fff;
-  color: #002529;
-}
-
-.buttonSecondaryHover:hover {
-  background-color: var(--secondary-hover-color);
-}
-
-.buttonSecondaryDisabled:disabled {
-  background-color: var(--secondary-disabled-color);
-}
-
-/* Tertiary Button */
-.buttonTertiary {
-  background-color: #fff;
-  color: var(--tertiary-color);
-  border: 2px solid var(--tertiary-color);
+  color: var(--button-color);
 }
 
-.buttonTertiaryHover:hover {
-  background-color: var(--tertiary-color);
-  color: var(--primary-white-color);
+.button:hover:enabled {
+  background-color: var(--button-hover-background-color);
+  color: var(--button-hover-color);
 }
 
-.buttonTertiaryDisabled:disabled {
-  color: var(--tertiary-disabled-color);
-  border: 2px solid var(--tertiary-disabled-border-color);
+.disabled {
+  background-color: #f0f0f0;
+  color: #ccc;
+  border-color: #ccc;
```

Comment:
use variables you added: 
`--primary-disabled-color: #666666;` (why is color on line 19 using disabled border color value?)
`--primary-disabled-border-color: #cccccc;`